### PR TITLE
Add doc to build only compiler component

### PR DESCRIPTION
**Pull request by dsouzai**

### CODE REVIEW

File: doc/build-instructions/Build_Compiler_Only.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+```
+EXTENSIONS_DIR=<path to openj9-openjdk extensions repo>
+JAVA_BASE=$EXTENSIONS_DIR/build/linux-x86_64-normal-server-release
+
+# Go to top level directory
+cd $EXTENSIONS_DIR/..
+
+# Set up directory structure
+ln -s $EXTENSIONS_DIR/openj9/runtime/compiler
```
Comment (andrewcraik) [inline doc/build-instructions/Build_Compiler_Only.md]: doesn't ln -s need two arguments?

Comment (dsouzai) [reply doc/build-instructions/Build_Compiler_Only.md]: No, if you just do `ln -s /long/path/to/dir/name` it'll create a symlink `name -> /long/path/to/dir/name`

**Comment by dsouzai** [comment]
@cathyzhyi do you mind also reviewing?

**Comment by cathyzhyi** [comment]
LGTM

**Comment by andrewcraik** [comment]
There is no code here - so build needed. Further, documentation looks good so will merge

**Pull request merged**